action.sh

```diff
@@ -24,7 +24,7 @@ if [ -d "${DEST}" ]; then
 fi
 
 echo '🍳 Build site'
-hugo -d ${DEST}
+hugo ${HUGO_ARGS} -d ${DEST}
```

Comment:
This script will fail on unset variables so I think you want this 

```suggestion
hugo ${HUGO_ARGS:-""} -d ${DEST}
```

This should use the empty string if `HUGO_ARGS` isn’t set (see https://stackoverflow.com/questions/12262696/using-unset-vs-setting-a-variable-to-empty#comment16445973_12263914)

Reply:
This variable is always set. Well I think it is. as it is defined in the work flow. And should not be removed?